### DISCUSSION TITLE
How do I prevent shortcuts in Chrome browsers?

### DISCUSSION THREAD

**Discussion by chl3188** [microsoft/monaco-editor#3897] [Q&A] [answered]
For example, if you register CtrlCmd+KeyT or CtrlCmd+KeyN with the addAction or addKeybindingRules function, the browser's function will perform instead of the command I defined.

Is there a way to block the browser's shortcut function when using a Monaco browser in a web environment?

**Comment by tamayika** [comment, marked as answer]
Several shortcuts as like you said cannot be disabled and you can not handle it. This is due to browser specification.
